### PR TITLE
Add PHPStan

### PR DESCRIPTION
**Pull request by holtkamp**

### CODE REVIEW

**Comment by holtkamp** [comment]
Shameless bump on this ;)

**Comment by l0gicgate** [comment]
If we're going to add phpstan we should add it to travis as well.

File: src/Application/Handlers/ShutdownHandler.php [hunk 1]
```diff
@@ -24,13 +24,6 @@ class ShutdownHandler
      */
     private $displayErrorDetails;
 
-    /**
-     * ShutdownHandler constructor.
-     *
-     * @param Request       $request
-     * @param $errorHandler $errorHandler
```
Comment (milescook) [inline src/Application/Handlers/ShutdownHandler.php]: Perhaps keep this doc block in, but to correct the doc and keep phpstan happy, change to:

`* @param HttpErrorHandler $errorHandler` 

I was about to create a PR but may as well piggy back off yours instead :)

Comment (holtkamp) [reply src/Application/Handlers/ShutdownHandler.php]: Sure, I was hoping for a "accept suggestion" button, but that seems not to be here...

**Comment by l0gicgate** [comment]
Resolved via #163

**Pull request closed without merging**